provider/pkg/provider/environment_test.go

```diff
@@ -14,14 +14,26 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-type getEnvironmentFunc func(ctx context.Context, orgName string, envName string, decrypt bool) (yaml []byte, etag string, err error)
+type getEnvironmentFunc func(ctx context.Context, orgName string, envName string, version string, decrypt bool) (yaml []byte, etag string, revision int, err error)
```

Comment:
What is the `version` here?

Reply:
This function has to match the signature of GetEnvironment method inside ESC client. I bumped the version and signature of the method changed, so had to change to match as well.